data/json/recipes/recipe_food.json

```
@@ -1483,7 +1513,7 @@
     "batch_time_factors": [ 80, 4 ],
     "qualities": [ { "id": "BOIL", "level": 1 } ],
     "tools": [ [ [ "water_boiling_heat", 2, "LIST" ] ] ],
-    "components": [ [ [ "wild_herbs", 10 ], [ "herbal_tea_bag", 1 ] ], [ [ "water", 1 ], [ "water_clean", 1 ] ] ]
+    "components": [ [ [ "wild_herbs", 10 ] ], [ [ "water", 1 ], [ "water_clean", 1 ] ] ]
```

```suggestion
    "components": [ [ [ "wild_herbs", 3 ] ], [ [ "water", 1 ], [ "water_clean", 1 ] ] ]
```
In addition, you will need to change the default calories of herbal tea to 27 in this mode.